spec/support/saml_auth_helper.rb

```diff
@@ -151,7 +155,7 @@ def saml_request
   end
 
   def key
-    @key ||= Rails.application.secrets.saml_client_private_key
+    @key ||= File.read("#{Rails.root}/config/saml_client_private_key.enc")
```

Comment:
Good catch, Sam. The SAML client key is really a _test_ SP private key and is only used by specs and the test controller.